addOns/commonlib/src/test/java/org/zaproxy/addon/commonlib/CookieUtilsUnitTest.java

```diff
@@ -274,6 +274,26 @@ void shouldReportNotExpiredIfEmptyHeaderOrInvalidCookieOrExpiresMissing(String h
         assertThat(expired, is(equalTo(false)));
     }
 
+    @ParameterizedTest
+    @ValueSource(
+            strings = {
+                "sessionId=38afes7a8",
+                "sessionId=a3fWa; Expires=Wed, 21 Oct 2015 07:28:00 GMT",
+                "sessionId=a3fWa; Max-Age=2592000",
+                "sessionId=219ffwef9w0f; Domain=somecompany.pt",
+                "sessionId=34d8g; SameSite=None; Secure; Path=/; Partitioned;",
+                "sessionId=value123; HttpOnly",
+                "sessionId=value123; Path=/lala/",
+            })
+    void shouldGetTheCorrectCookieName(String headerValue) {
+        // Given
+        String header = headerValue;
+        // When
+        String name = CookieUtils.getCookieName(header);
```

Comment:
```suggestion
       // Given / When
       String name = CookieUtils.getCookieName(headerValue);
```

Reply:
Agree, but the other tests did it this way. Done.